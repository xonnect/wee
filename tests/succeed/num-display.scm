(display 32768)
(display " -> 32768\n")
(display 1000000)
(display " -> 1000000\n")
(display -32768)
(display " -> -32768\n")
(display -70000)
(display " -> -70000\n")
(display (+ 32768 -32768))
(display " -> 0\n")
