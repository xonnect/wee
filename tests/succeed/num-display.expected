32768 -> 32768
1000000 -> 1000000
-32768 -> -32768
-70000 -> -70000
0 -> 0
